pkg/driver/driver.go

```diff
@@ -13,7 +13,9 @@ type Driver interface {
 
 	CreateDisk() error
 
-	Start(_ context.Context) (chan error, error)
+	Start(_ context.Context) (bool, chan error, error)
```

Comment:
Please add a comment to explain bool

Reply:
Done, Added comments for start and stop methods in driver. Do let me know if you have some suggestions on those comments